.vscode/settings.json

```diff
@@ -4,30 +4,21 @@
   "javascript.validate.enable": false,
   "css.validate": false,
   "scss.validate": false,
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "editor.formatOnSave": true,
   "typescript.tsdk": "./node_modules/typescript/lib",
   "flow.enabled": false,
   "files.exclude": {
     "**/.git": true,
-    "**/.svn": true,
-    "**/.hg": true,
     "**/node_modules": true,
     "**/.DS_Store": true,
-    "build": true,
-    ".dev": true
+    "packages/*/{.sewing-kit,build}": true,
+    ".{dev,shadowenv.d,sewing-kit}": true
   },
   "files.insertFinalNewline": true,
   "files.trimTrailingWhitespace": true,
-  "search.exclude": {
```

Comment:
search.exclude grandfathers in all the patterns in files.exclude so we don't need to duplicate them.